CHANGELOG.md

```diff
@@ -806,3 +806,10 @@ Initial version to have a stable reference before [gossip protocol removal](/doc
 - Bucket commands.
 - Downsampling support for UI.
 - Grafana dashboards for Thanos components.
+
+
+### Added
+
+- Search box functionality on the Thanos website.
+
+## [#2664 ](https://github.com/thanos-io/thanos/pull/2664) - 2020.5.26
```

Comment:
I think you added the Changelog in the wrong place. You need to add the changelog at the top in the unreleased section.

Reply:
Oh...I think that might be the issue. Would fix that now